EIPS/eip-1789.md

```diff
@@ -33,12 +33,14 @@ We recognize that, while technically trivial, the real challenge in inflation-ba
 
 Two new constants are introduced: BENEFICIARY_ADDRESS, an Address, and DEVFUND_BLOCK_REWARD, an amount denominated in wei. Both are set to zero.
 
-Beginning with block ISTANBUL_BLOCK_HEIGHT, a coinbase transaction is added to each block minting DEVFUND_BLOCK_REWARD wei and adding it to the balance of BENEFICIARY_ADDRESS.
+Beginning with block ISTANBUL_BLOCK_HEIGHT, if DEVFUND_BLOCK_REWARD is greater than zero, a coinbase transaction is added to each block minting DEVFUND_BLOCK_REWARD wei and adding it to the balance of BENEFICIARY_ADDRESS.
```

Comment:
I don't think any transaction is actually generated here (or for existing block rewards). I think the coinbase account address is just directly credited in the state, which is different from how Bitcoin works. I need to take another look t this.

Reply:
ok